package.json

```diff
@@ -49,9 +53,11 @@
     "lint-staged": "^10.5.3",
     "node-fetch": "^2.6.1",
     "prettier": "2.1.2",
+    "rxjs": "^7.5.5",
     "testcontainers": "^8.10.1",
     "ts-jest": "^26.4.4",
     "ts-node": "9.0.0",
-    "typescript": "4.0.3"
+    "typescript": "^4.6.2",
+    "ws": "^8.5.0"
```

Comment:
Not related but imo we should enforce fixed versions to prevent attacks via the build system...

Reply:
Created an issue to talk about this https://github.com/linagora/jmap-client-ts/issues/69